jupyterlab/labextensions.py

```diff
@@ -158,11 +164,15 @@ def run_task(self):
 
 class UninstallLabExtensionApp(BaseExtensionApp):
     description = "Uninstall labextension(s) by name"
+    flags = uninstall_flags
+
+    all = Bool(False, config=True,
+        help="Whether to uninstall all extensions")
 
     def run_task(self):
         self.extra_args = self.extra_args or [os.getcwd()]
         return any([
-            uninstall_extension(arg, self.app_dir, logger=self.log)
+            uninstall_extension(arg, self.all, self.app_dir, logger=self.log)
```

Comment:
After changing the signature, this will now fail (and this part is not covered by the test). I think this is the final bit :)

Reply:
Sorry about that again ! I'll be more careful. Made the arguments explicit and squashed the commits